vendor.conf

```diff
@@ -124,3 +124,7 @@ golang.org/x/time f51c12702a4d776e4c1fa9b0fabab841babae631
 github.com/cri-o/ocicni 84aa158d2bacf95147b2b0a89615dd665630f440
 github.com/projectatomic/libpod master
 github.com/containerd/continuity d8fb8589b0e8e85b8c8bbaa8840226d0dfeb7371
+github.com/projectatomic/buildah cf753ee6fe1f606c4456d2d76690534f8170d9a3
+github.com/openshift/imagebuilder master
+github.com/fsouza/go-dockerclient master
+github.com/Nvveen/Gotty master
```

Comment:
These are strange dependencies for the commands you're implementing.  Based on some naive greps, I don't see anybody consuming github.com/openshift/imagebuilder.  Can we drop that?  Once we drop that, I don't see anybody else consuming github.com/fsouza/go-dockerclient.  Can we drop that too?  Once we drop that, I don't see anybody else consuming github.com/Nvveen/Gotty.  Can we drop that too?  And (possibly because I removed the earlier packages), I don't see anybody consuming github.com/projectatomic/buildah.  So basically I think we can drop this whole hunk and save ourselves 84 vendored files with no loss of functionality.

Reply:
Can't speak for the last two, but I'm pretty sure buildah is sucking down imagebuilder.  I'm not sure if libpod also needs to do so, it may not.

Reply:
I know libpod as a whole does consume buildah, but I don't know if the image library specifically requires it.

Reply:
On IRC, @baude pointed out [this use][1].  We're trying to see if we can avoid the dep there...

[1]: https://github.com/projectatomic/libpod/blob/99959e55fa8e902302a580b13d53bc52d0074ad7/libpod/image/image.go#L943

Reply:
> On IRC, @baude pointed out this use. We're trying to see if we can avoid the dep there...

Fix in flight with projectatomic/libpod#1054.

Reply:
fixed

Reply:
I still see these in the PR.